src/main/resources/assets/api-docs.yaml

```diff
@@ -2049,6 +2049,51 @@ paths:
           description: The requested user doesnt exist in the system
           schema:
             $ref: '#/definitions/ErrorResponse'
+  /api/nih-register/{userId}:
```

Comment:
This doesn't match `NihAccountResource` any more

Reply:
This is now updated.